app/src/main/java/org/sopt/and/ui/screen/signUp/composable/SignUpScreen.kt

```diff
@@ -0,0 +1,292 @@
+package org.sopt.and.ui.screen.signUp.composable
+
+import android.widget.Toast
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.focus.onFocusChanged
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import org.sopt.and.R
+import org.sopt.and.ui.components.textField.TextFieldCustom
+import org.sopt.and.ui.screen.signUp.viewmodel.SignUpViewModel
+import org.sopt.and.ui.theme.BlueBtnColor
+import org.sopt.and.ui.theme.darkGray1
+import org.sopt.and.ui.theme.darkGray3
+import org.sopt.and.ui.theme.darkGray4
+import org.sopt.and.ui.theme.errorColor
+
+@Composable
+fun SignUpScreen(
+    navController: androidx.navigation.NavHostController,
```

Comment:
테스트 용이성이나 단일 책임원칙 등에 의거하여, 하위 Composable에 navController 자체를 전달하기보다는, navController를 가지고 수행할 액션을 전달하는 것을 권장하고 있습니다 !!
```suggestion
    onNavigateToSignIn: () -> Unit
```
이런식으루요 !

Reply:
참고해서 수정하겠습니다! 감사합니다!